maint/precision/compare_ops.py

```
@@ -0,0 +1,428 @@
+            }
+        )
+        out = kernel(x)
+
+    # Restore original shape
+    return out.view(original_shape)
+
+def triton_op(x: torch.Tensor, op_id: int, y: Optional[torch.Tensor] = None) -> torch.Tensor:
+    """Standard Triton operation interface."""
+    assert x.is_cuda
+    out = torch.empty_like(x)
+    grid = lambda meta: ((x.numel() + meta['BLOCK_SIZE'] - 1) // meta['BLOCK_SIZE'],)
```

Ruff (0.13.1)

Do not assign a `lambda` expression, use a `def`

Rewrite `grid` as a `def`

(E731)